apache2/msc_xml.c

```diff
@@ -132,29 +132,19 @@ static void msc_xml_on_characters(void *ctx, const xmlChar *ch, int len) {
 }
 
 
-static xmlParserInputBufferPtr
```

Comment:
Have you removed this function because it always returned a NULL?

Reply:
Ah, this is a mistake too. Since `entity` was set but never used in xml_init I assumed this was safe to remove, but `xmlParserInputBufferCreateFilenameDefault` should still be called here. I'll restore it and fix the warning another way.

Reply:
Removed this change in 9d9a72734988bbd4f101cbea0d5df60eabcc1435